### ISSUE TITLE
Unable to use 'slack send'

### ISSUE THREAD

**Issue by judepereira** [microservice/slack#15]
The send action results in the following error:

```
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://0.0.0.0:8000/ (Press CTRL+C to quit)
[2019-02-20 16:28:38,331] ERROR in app: Exception on /api/chat.PostMessage [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/app/api.py", line 17, in api
    endpoint, **prepare(request.json)
  File "/usr/local/lib/python3.7/site-packages/slackclient/client.py", line 184, in api_call
    self.token, request=method, timeout=timeout, **kwargs
TypeError: api_call() got multiple values for argument 'token'
```

**Comment by stevepeak** [comment]
@judepereira what curl request did we can reproduce this 👍

**Comment by judepereira** [comment]
Does it work via the omg cli?

**Comment by stevepeak** [comment]
Fixed in b59e6a2fbf1b32e7e7077543c690491c6a77825b

**Comment by judepereira** [comment]
Reproduce this with:
```
$ curl -d '{"limit": null, "token": null, "types": null, "exclude_archived": null}' -H "Content-Type: application/json; charset=utf-8" http://slack-54f44141f42ea7503ad2e6e134e55a436a8dbcb5.a677e1f4-eba6-4943-8c5e-c98ba3e0bc45.svc.cluster.local:8000/api/channels.list
```

Logs from the Slack service:
```
[2019-02-23 05:28:53,204] ERROR in app: Exception on /api/channels.list [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/app/api.py", line 17, in api
    endpoint, **prepare(request.json)
  File "/usr/local/lib/python3.7/site-packages/slackclient/client.py", line 184, in api_call
    self.token, request=method, timeout=timeout, **kwargs
TypeError: api_call() got multiple values for argument 'token'
10.60.130.3 - - [23/Feb/2019 05:28:53] "POST /api/channels.list HTTP/1.1" 500 -

```
